Rubberduck.VBEEditor/Extensions/VbProjectExtensions.cs

```diff
@@ -1,3 +1,104 @@
-﻿namespace Rubberduck.VBEditor.Extensions
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+
+namespace Rubberduck.VBEditor.Extensions
 {
+    public static class VBProjectExtensions
```

Comment:
At first I thought killing the extension methods by moving them to the wrapper types would be a good idea, but now I think the wrapper methods that need to exist in both VBA and VB6 implementations are best implemented as extension methods (extending the interfaces as you did) - that way we won't need copy-pasta or double inheritance 👍